### PR TITLE
Update bug report template

### PR DESCRIPTION
**Pull request by ghost1372**
Untitled

### CODE REVIEW

**Comment by ghost1372** [comment]
Hi @andrewleader 
Can you review?

**Comment by jonwis** [comment]
What do the issues generated by this new format look like?

**Comment by andrewleader** [comment]
Whoa this is super cool! I didn't know this was possible @ghost1372, thanks for opening the pull request! We'll review and work on this some throughout the week.

**Comment by ghost1372** [comment]
> What do the issues generated by this new format look like?

Untitled

**Comment by ghost1372** [comment]
> Whoa this is super cool! I didn't know this was possible @ghost1372, thanks for opening the pull request! We'll review and work on this some throughout the week.

Thanks
Checkbox template is also available if needed
ss

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -55,6 +55,13 @@ body:
         - "Microsoft.ProjectReunion 0.8.2"
         - "Microsoft.ProjectReunion 0.8.1"
         - "Microsoft.ProjectReunion 0.8.0"
+  - type: checkboxes
+    attributes:
+      label: Project type
+      description: Please specify project type you have encountered the issue.
+      options:
+        - label: "Packaged"
+        - label: "UnPackaged"
```
Comment (andrewleader) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:64]: UnPackaged shouldn't have a capital P, should be just `Unpackaged`

Comment (ghost1372) [reply .github/ISSUE_TEMPLATE/bug-report.yaml]: Done

**Review by andrewleader — APPROVED** [review]
This looks good to me! @jonwis any remaining questions/concerns?

**Review by axelandrejs — APPROVED** [review]

**Pull request merged**